### PR TITLE
Use the README template

### PR DESCRIPTION
**Pull request by MatMoore**
Use the recommended template from https://github.com/alphagov/styleguides/blob/master/use-of-READMEs.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,16 @@
...
 This is a gem to share code between GOV.UK frontends.
 
-## Installation
+## Nomenclature
+
+- **content item**: An object returned by the content store
+
+## Technical documentation
+
+These helpers present data from the [content store](https://github.com/alphagov/content-store) for use with [slimmer](https://github.com/alphagov/slimmer) to render common components on GOV.UK.
```
Comment (tijmenb) [inline README.md]: slimmer -> govuk_components ?

File: README.md [hunk 2]
```diff
@@ -14,6 +23,35 @@ And then execute:
 
     $ bundle
 
-## Usage
+Or install it yourself as:
```
Comment (tijmenb) [inline README.md]: We can skip the "install it yourself" step, nobody uses that.

File: README.md [hunk 3]
```diff
@@ -14,6 +23,35 @@ And then execute:
...
+```
+
+### Running the test suite
+
+`bundle exec rake`
+
+## Licence
 
-TBD.
+[MIT License](LICENCE.txt)
```
Comment (tijmenb) [inline README.md]: It's `LICENSE.txt` (which should be changed to our default license btw)

Comment (MatMoore) [inline README.md]: I think licenCe would be correct here and licenSe is the verb (stupid english language)

I'm updating the file to match our other libraries.

**Comment by tijmenb** [comment]
@MatMoore can you rebase this?

**Pull request merged**